### PR TITLE
refactor: credential display fields

### PR DESCRIPTION
**Pull request by Norlock**
# Description of change
On multiple places multiple sources are used for the display name, this PR moves that logic to the BE and are directly placed so it can always use that field. It also removes two fields which aren't used in the FE. This PR should wait until dev-mode is merged.

## How the change has been tested
I ran the Dragon profile and made sure the before and after returned the same results.

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: identity-wallet/src/state/reducers/mod.rs [hunk 1]
```diff
@@ -116,7 +116,7 @@ pub async fn create_identity(state: AppState, action: Action) -> Result<AppState
         let profile = Profile {
             name: name.to_string(),
             picture: Some(picture.to_string()),
-            theme: Some(theme.to_string()),
+            theme: Some(theme),
```
Comment (nanderstabel) [inline identity-wallet/src/state/reducers/mod.rs:119]: Not sure why we're using `to_string` here :sweat_smile: Can you also remove it for `name` and `picture`?

Comment (Norlock) [reply identity-wallet/src/state/reducers/mod.rs:119]: Ok top ik voeg deze changes toe aan de nieuwe PR goed? dan is het wat makkelijker

Comment (nanderstabel) [reply identity-wallet/src/state/reducers/mod.rs:119]: Sure :ok_hand:

**Comment by Norlock** [comment]
Will be done in: #139

**Pull request closed without merging**